cmd/ore/aws/upload.go

```diff
@@ -252,6 +269,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 				os.Exit(1)
 			}
 		}
+
+		if len(uploadTags) > 0 {
```

Comment:
The logic here makes sense to me **but** it looks identical to the previous block. Maybe this should become a function?

Reply:
agreed.